### PR TITLE
Make it work

### PR DESCRIPTION
**Pull request by baronfel**
so the bad stuff was that I had to manually make a change to portacode to let it run under netcoreapp, so keep an eye on that.

### CODE REVIEW

File: src/HotReload.Client/Main.fs [hunk 1]
```diff
@@ -10,22 +10,22 @@ type Model =
...
     | Decrement -> { model with value = model.value - 1 }
 
-let view model dispatch =
+let initModel =
+    {
+        value = 0
+    }
+
+
+let mutable view = fun model dispatch ->
```
Comment (baronfel) [inline src/HotReload.Client/Main.fs:28]: probably don't need view and update as mutable anymore

File: src/HotReload.Listener/ListenerController.fs [hunk 2]
```diff
@@ -13,5 +61,6 @@ type ListenerController(hub : IHubContext<ReloadHub>) =
     member this.Update () =
         let reader = new StreamReader(this.Request.Body)
         let json = reader.ReadToEnd ()
-        printfn "Received %s" json
-        hub.Clients.All.SendAsync("Update", json)
\ No newline at end of file
+        let files = JsonConvert.DeserializeObject<(string * DFile)[]>(json)
+        Interpreter.handleUpdate files
+        //hub.Clients.All.SendAsync("Update", json)
```
Comment (baronfel) [inline src/HotReload.Listener/ListenerController.fs:66]: could uncomment this to both do the work serverside and send to the client

File: src/HotReload.Listener/Startup.fs [hunk 3]
```diff
@@ -11,7 +11,7 @@ type Startup() =
     // This method gets called by the runtime. Use this method to add services to the container.
     // For more information on how to configure your application, visit https://go.microsoft.com/fwlink/?LinkID=398940
     member this.ConfigureServices(services: IServiceCollection) =
-        services.AddMvc().SetCompatibilityVersion(CompatibilityVersion.Version_2_2) |> ignore
+        services.AddMvc().SetCompatibilityVersion(CompatibilityVersion.Version_2_1) |> ignore
```
Comment (baronfel) [inline src/HotReload.Listener/Startup.fs:14]: had to use netcoreapp2.1 for some reason, which is why this flipped back

Comment (tylerhartwig) [reply src/HotReload.Listener/Startup.fs]: > so the bad stuff was that I had to manually make a change to portacode to let it run under netcoreapp, so keep an eye on that.

Was this just skipping over the exception on loading `System.IO.Pipelines` or did you have to make other changes?

**Comment by tylerhartwig** [comment]
Closing as I've moved to using FCSWatch

**Pull request closed without merging**